docs-src/security-model.md

```diff
@@ -0,0 +1,148 @@
+* **Application developer**: The application developer (dev) is responsible for
+  defining their application configuration (e.g. timeouts, request
+  matching/filter) and Service composition (e.g. path routing to backends). 
+
+Although these roles can cover a wide variety of use cases, some organizations
+may be structured slightly differently. Many organizations may also have a
+fourth role that sits between "cluster operator" and "application developer":
+
+* **Application admin**: The application admin has administrative access to some
+  namespaces within a cluster, but not the cluster as a whole.
+
+## The Security Model
+There are two primary components to the Service APIs security model: RBAC and
+namespace restrictions.
+
+## RBAC
+RBAC (role-based access control) is the standard used for Kubernetes
+authorization. This allows users to configure who can perform actions on
+resources in specific scopes. RBAC can be used to enable each of the roles
+defined above. In most cases, it will be desirable to have all resources be
+readable by most roles, so instead we'll focus on write access for this model.
+
+### Write Permissions for Simple 3 Tier Model 
+| | GatewayClass | Gateway | Route | TrafficSplit |
+|-|-|-|-|-|
+| Infrastructure Provider | Yes | Yes | Yes | Yes |
+| Cluster Operators | No | Yes | Yes | Yes |
+| Application Developers | No | No | Yes | Yes |
+
+### Write Permissions for Advanced 4 Tier Model 
+| | GatewayClass | Gateway | Route | TrafficSplit |
+|-|-|-|-|-|
+| Infrastructure Provider | Yes | Yes | Yes | Yes |
+| Cluster Operators | Sometimes | Yes | Yes | Yes |
+| Application Admins | No | In Specified Namespaces | In Specified Namespaces | In Specified Namespaces |
+| Application Developers | No | No | In Specified Namespaces | In Specified Namespaces |
+
+## Namespace Restrictions
+The extra configuration options are not possible to control with RBAC. Instead,
+they will be controlled with configuration fields on GatewayClasses:
+
+* **gatewayNamespaceSelector**: This field is a selector of valid namespaces
+  that Gateways can use this GatewayClass from. An empty selector (default)
+  indicates that Gateways can use this GatewayClass from any namespace.
+
+* **routeNamespaceSelector**: This field is a selector of valid namespaces that
+  gateways can reference routes from. An empty selector (default) indicates that
+  routes can be referenced from any namespace.
+
+## Controller Requirements
+To be considered conformant with the Service APIs spec, controllers need to:
+
+* Populate status fields on Gateways and Resources to indicate if they are
+  compatible with the corresponding GatewayClass configuration.
+* Not implement invalid configuration. If a route is referenced in an invalid
+  namespace for the GatewayClass, it should be ignored. If a route has TLS
+  configuration and the GatewayClass does not allow TLS configuration on routes,
+  the TLS configuration on that route should be ignored.
```

Comment:
Nitpick on TLS: Ignoring TLS settings and then satisfying a route can get vague and confusing. Interpretation of a route definition could change significantly if TLS is ignored and impact other routing policies. Does ignoring the route completely if TLS is specified when it should not be make sense? We can start small and open things up in future.

Reply:
Good catch, I'll remove that example entirely, since TLS config on alternate resources has been removed from the scope of this doc.